packages/openapi3/package.json

```diff
@@ -44,15 +47,21 @@
     "test:ci": "vitest run --coverage --reporter=junit --reporter=default",
     "lint": "eslint . --max-warnings=0",
     "lint:fix": "eslint . --fix",
-    "regen-docs": "tspd doc .  --enable-experimental  --output-dir ../../docs/emitters/openapi3/reference"
+    "regen-docs": "tspd doc .  --enable-experimental  --output-dir ../../docs/emitters/openapi3/reference",
+    "generate-tsp": "node --no-warnings --experimental-specifier-resolution=node --loader ts-node/esm ./test/openapi-to-typespec/utils/generate-typespec.ts",
+    "gen-version": "node scripts/generate-version.js"
   },
   "files": [
     "lib/*.tsp",
     "dist/**",
     "!dist/test/**"
   ],
   "dependencies": {
-    "yaml": "~2.4.2"
+    "@readme/openapi-parser": "~2.6.0",
+    "@typespec/prettier-plugin-typespec": "workspace:~",
+    "prettier": "~3.2.5",
+    "yaml": "~2.4.2",
+    "yargs": "~17.7.2"
```

Comment:
think with how simple this cli should be we could just use `parseArgs` from node

Reply:
Little hesitant to do this while we still support node.js 18 since `parseArgs` only became stable in node.js 20. How strongly do you feel about this?

Reply:
hhm, we already use it in a few of the other eng sys tool so I think it should be good enough and not have us needing to cleanup. Just some extra dependency that gets bundled in the playground too

Reply:
Fair enough - updating!